test.js

```diff
@@ -148,5 +148,24 @@ assert.deepStrictEqual(
   }
 );
 
+assert.deepStrictEqual(
+  editorconfigToPrettier({ insert_final_newline: false }),
+  { insertFinalNewline: false }
+);
+
+assert.deepStrictEqual(
+  editorconfigToPrettier({ insert_final_newline: "false" }),
+  {}
+);
```

Comment:
Do you think we need this test at all? Not sure how realistic this scenario is...

Reply:
I wanted to test an invalid value (wrong type here). But it might be well unnecessary. :)

Reply:
I added a commit to remove the unecessary test scenarios.

Reply:
Oh, I see. I think that since this function expects an already-parsed editorconfig object, it's safe to assume that the values will be valid, so I don't want to worry about testing invalid values. Thanks for removing the extra tests.